snuba/cli/migrations.py

```diff
@@ -71,11 +73,13 @@ def migrate(
     runner = Runner()
 
     try:
-        if group is not None:
+        if group != "all":
```

Comment:
nit: maybe its just me, but I'd rather use `group=None` to signal that all migrations should be run rather than `group='all'` makes it clear that its the default special case by having a null type, and avoid any bugs if we ever decide to make a group named 'all'

Reply:
@dbanda It seems like to me that positional arguments don't work if you have the default be `None`, I get 

`Error: Missing argument 'GROUP'.` 

So if we want it to be an option I can change it, but I was trying to make it so you didn't have to type like `-g` or `--group`.  What are your thoughts?